paper/paper.md

```diff
@@ -0,0 +1,188 @@
+modeling tools.
+The library is supported by extensive, regularly updated
+documentation, including examples (in particular, a Getting Started
+example), various use cases, tutorials, and user guides.
+The documentation makes the library very user-friendly.
+OpenTURNS integrates seamlessly with NumPy, Pandas, and other
+scientific Python packages, allowing users to develop UQ
+workflows and generate reproducible results.
+Its modular architecture enables both newcomers and advanced users to
+access reliable, well-tested methods for uncertainty propagation,
+sensitivity analysis, and reliability assessment.
+Its active community provides additional support and resources through 
+a Discourse forum and its website.
+
+# Statement of need
+
+UQ plays a critical role in robust decision-making during the design
+and operation of complex systems.
+It encompasses probabilistic modeling (multivariate distributions,
+stochastic processes), sampling, functional algebra, uncertainty
+propagation, sensitivity analysis, reliability, optimization, inverse
+problems, and surrogate modeling.
+Validated, well-tested and well-documented methods are essential for teaching, academic research, and industrial applications.
+However, many probabilistic modeling libraries are domain-specific, 
+focusing on fields such as mechanical engineering, computational fluid
+dynamics, or hydrology, which limits the sharing of best practices.
+This underscores the need for a general-purpose, cross-disciplinary
+framework for probabilistic modeling.
+
+Of course, other Python packages exist for Monte-Carlo
+sampling and other numerical methods [@virtanen2020scipy],
+statistical modeling [@seabold2010statsmodels], 
+machine learning [@pedregosa2011scikit], for providing UQ test 
+functions [@Wicaksono2023], and for quasi-Monte Carlo methods [@roy2023quasi]
+(among others).
+One of the goals of OpenTURNS is to provide a consistent interface to 
+these tasks in the UQ context.
+
+# Features
+
+OpenTURNS provides a wide range of multivariate probabilistic modeling
+features: more than 60 continuous and discrete distributions, 
+including extreme value distributions, copula-based models (more than 
+15 copulas), and distribution transformations: truncation, conditioning, mixture, push-forward and distribution algebra, etc.
+User-defined distributions can be easily implemented and integrated
+natively.
+
+Multivariate distributions can be sampled using general sampling algorithms
+(Monte-Carlo, Latin Hypercube Sampling - LHS, and optimized LHS) and quasi–Monte Carlo
+low-discrepancy sequences (such as Sobol’ or Halton) for integration
+and model analysis.
+
+OpenTURNS offers modeling features for stationary 
+and non-stationary stochastic processes (ARMA processes, Gaussian 
+processes in arbitrary dimensions) associated with several covariance 
+kernels (e.g., exponential, Matérn, and squared exponential).
+
+The library provides extensive data analysis capabilities
```

Comment:
Issue with the sentence: 
The library provides extensive data analysis capabilities, including the ability to compute dependency indicators

Reply:
Oups. Ok, I will do this.

Reply:
You were right: the paragraph was not correctly written. This is fixed.